corehq/apps/app_manager/serializers.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from rest_framework import serializers
+
+from corehq.apps.app_manager.models import (
+    LatestEnabledBuildProfiles,
+)
+
+
+class LatestEnabledBuildProfileSerializer(serializers.ModelSerializer):
+    class Meta(object):
+        model = LatestEnabledBuildProfiles
+        fields = ['id', 'app_id', 'active', 'version']
+
+    def to_representation(self, instance):
+        ret = super(LatestEnabledBuildProfileSerializer, self).to_representation(instance)
+        build_profile_id = self.instance.build_profile_id
+        profile = self.instance.build.build_profiles.get(build_profile_id)
+        profile_name = profile.name if profile else build_profile_id
+        ret['app_name'] = self.context['app_names'][ret['app_id']]
```

Comment:
you mean `build.name`? I did think about it. 
`build.name` would give me the name of the app when the build was created and its possible that has changed now and it would get confusing to see a name in the list which is not in current apps list. `app_names` would give current names.